src/Engine/UnitActionLifecycles/BattleLifecycle.java

```diff
@@ -191,12 +192,14 @@ public GameEventQueue getEvents(MapMaster gameMap)
       BattleSummary summary = CombatEngine.simulateBattleResults(attacker, defender, map,
                               moveCoord.xCoord, moveCoord.yCoord);
 
+      int attackerHPLoss = (int) (-10 * summary.attacker.deltaPreciseHP);
+      int defenderHPLoss = (int) (-10 * summary.defender.deltaPreciseHP);
```

Comment:
Is this always accurate? If an attack does 1.7 damage, reducing a unit from 5.3 to 3.4 health, then the precise delta of 1.7 would be an HP change of 2 (6->4).

Reply:
I guess it's the same calculation we did before, so it's at least not more broken than it was.

Reply:
Changed to *HealthLoss in PR2